build.gradle

```diff
@@ -41,7 +41,7 @@ dependencies {
     testImplementation ('ch.qos.logback:logback-classic:1.2.11')
 }
 
-ext.compatibilityVersion = JavaVersion.VERSION_11
+ext.compatibilityVersion = JavaVersion.VERSION_1_8
```

Comment:
Unfortunately, I did not see this. This change breaks javadoc. Why downgrade Java here?

Reply:
@martinda internal requirement here to use an older version. Code still works as expected and compiles correctly. Any issues you can see from your end? If we need to change things back I'm open for discussion.

Reply:
You saw my fix, and thanks for merging it so quickly!